README.md

```diff
@@ -136,6 +136,14 @@ Set flash mode (`auto`/`on`/`off`)
 const success = await this.camera.setFlashMode(newFlashData.mode);
 ```
 
+#### setTorchMode - must have the wanted camera capture reference
+
+Set Torch mode (`auto`/`on`/`off`)
+
+```js
+const success = await this.camera.setTorchMode(TORCH_MODE_ON);
```

Comment:
What is the `TORCH_MODE_ON`? 
The API is getting one of:
`auto`/`on`/`off`

Reply:
fixed